File: 2529-maximum-count-of-positive-integer-and-negative-integer/2529-maximum-count-of-positive-integer-and-negative-integer.cpp
class Solution {
public:
    int maximumCount(vector<int>& nums) {
        int cntPlus = 0, cntNeg = 0;
        for(int &num : nums) {
            if(num < 0) cntNeg++;
            else if(num > 0) cntPlus++;
        }
        return max(cntPlus, cntNeg);
    }
};

File: 2529-maximum-count-of-positive-integer-and-negative-integer/2529-maximum-count-of-positive-integer-and-negative-integer.py
class Solution:
    def maximumCount(self, nums: List[int]) -> int:
        cntPlus = cntNeg = 0
        for num in nums:
            if num < 0: cntNeg += 1
            elif num > 0: cntPlus += 1
        return max(cntPlus, cntNeg)